standalone-metastore/pom.xml

```diff
@@ -181,6 +182,17 @@
         <artifactId>commons-lang3</artifactId>
         <version>${commons-lang3.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.datasketches</groupId>
```

Comment:
I think this has nothing to do with this patch

Reply:
why do you need this, it's already defined in root pom DM?

Reply:
`standalone-metastore` does not inherit from the root pom